src/HIndent/Pretty.hs

```diff
@@ -922,7 +922,7 @@ instance Pretty Asst where
            write " ~ "
            pretty b
       ParenA _ asst -> parens $ pretty asst
-      VarA _ var -> pretty var
+      AppA _ var _ -> pretty var
```

Comment:
I don't think this can be right. Surely the third argument also needs to be printed.